### PR TITLE
Makes the language selection dropdown visible on mobile view

### PR DESCRIPTION
**Pull request by SomtochiAma**
The language drop down isn't visible on mobile view.

Screenshot (32)

The pr makes it visible on mobile view
Screenshot (33)

### CODE REVIEW

File: dashboard/views/login.ejs [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-                  <option value="en">English</option>
-                  <option value="es">Spanish</option>
-                  <option value="fr">French</option>
-                  <option value="hi">Hindi</option>
-                </select>
-            </div>
+        <div class="navbar-form navbar-right pull-right">
+            <select class="form-control" id="languageSelection">
+              <option value="en">English</option>
+              <option value="es">Spanish</option>
```
Comment (tarunsinghal92) [inline dashboard/views/login.ejs]: formatting looks little messed. please rectify.

Comment (SomtochiAma) [reply dashboard/views/login.ejs]: Let me work on that.

Comment (SomtochiAma) [reply dashboard/views/login.ejs]: I just made a new commit with the code formatted.

**Comment by SomtochiAma** [comment]
Yes I can, I will work on it

**Comment by SomtochiAma** [comment]
I have updated the code so there is no conflict.

**Comment by llaske** [comment]
Nice.

**Pull request merged**